Makefile

```diff
@@ -1,4 +1,4 @@
-CONTAINERS = grafana graphite-statsd logstash monitoring-proxy sensu sensu-api sensu-client sensu-server uchiwa
```

Comment:
Are we keeping logstash code, but not producing any more images?
I would have though logstash folder would be deleted too.